docs/reportdaily.rst

```diff
@@ -48,12 +48,95 @@ Subcommands
 init
 ~~~~
 
-Is required for the first time use of the programm
-User can enter his data directly or he need to answer some questions provided by this command
+The ``init`` subcommand is required for the first time to initialize the program.
 
-.. code:: bash 
+The user is asked to answer some question. After the input is finished, two files are created:
+
+A configuration file with all the user information and a sqlite database to hold all data.
+
+
+.. code-block:: bash 
 
    reportdaily init
+   reportdaily init  --help
+   reportdaily init  --change [--duration DURATION] [--name Name][--team TEAM] [--team-number TEAM_NUMBER] [--year YEAR]
+
+.. option:: --change, -c
+
+Make changes to the existing config/database
+
+Use the option  `--change` in two different  ways:
+
+ 
+1. Ask for user input
+
+   .. code-block:: bash
+            reportdaily init --change
```

Comment:
Sphinx/RST needs some (empty) lines to distinguish between the directive ([`.. code-block::`](https://www.sphinx-doc.org/en/1.4.9/markup/code.html#directive-code-block)) and potential parameters for this directive.

If you don't separate your content, Sphinx will interpret your next line as option. As `reportdaily ...` is an unknown option, you will get an error. As such, you need:

```suggestion
   .. code-block:: bash

       reportdaily init --change
```